tests/cleanup/remove_old_objects_azure.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+# Script removes virtual machines and other artifacts older than HOURS_LIMIT (24 hours by default) from Azure
+
+. /usr/share/beakerlib/beakerlib.sh
+
+
+# Delete old objects based on the $TAG_NAME tag value defined in a previous execution of the script
+delete_old_resources() {
+    local resource_type="$1"
+
+    # list resources older than $TIMESTAMP based on the $TAG_NAME tag created in a previous run
```

Comment:
Doesn't that mean the script will tag new resources today and delete them tomorrow, essentially leaving them hanging 2× the defined time ?

Current approach is good (manually created stuff) but adding the tagging when the resources are consumed from the test script will be even better, more optimal.

Reply:
Since the cleaning script is executed every hour, the new resource will be removed in ~25 hours.
Extending the test script to add tags for VMs should not be difficult, but tagging other resources (ip addresses, network security groups,...) would require an extra effort to implement.

Reply:
VM tagging by the test is fine. The biggest cost on Azure is compute time and then storage. When we had the spike in cost the stats show network badwidth but I can't find more info about that in Azure's reports. Go figure.